lib/improver/blending/weighted_blend.py

```diff
@@ -113,6 +113,8 @@ def conform_metadata(
                 forecast_period_coord(cube,
                                       force_lead_time_calculation=True))
             forecast_period.bounds = None
+            forecast_period.convert_units(cube.coord("forecast_period").units)
+            forecast_period.var_name = cube.coord("forecast_period").var_name
```

Comment:
I guess this fixed the bug I was seeing when running the suite, so hooray.

Reply:
Yep!